fix for SWIG existing with 'exit 1' when "SWIGing tensorflow/python/tensorflow.i"
see also https://github.com/bazelbuild/bazel/issues/4053
author: Kenneth Hoste (HPC-UGent)
--- tensorflow/tensorflow.bzl.orig	2018-02-28 19:52:31.000000000 +0100
+++ tensorflow/tensorflow.bzl	2018-03-13 14:01:10.665060503 +0100
@@ -1044,6 +1044,7 @@
   args += [src.path]
   outputs = [ctx.outputs.cc_out, ctx.outputs.py_out]
   ctx.action(
+      use_default_shell_env = True,
       executable=ctx.executable._swig,
       arguments=args,
       inputs=list(inputs),
